test/test_im_onix.rb

```diff
@@ -833,4 +833,18 @@ def test_products_discount
       assert_equal nil, prices[4][:until_date]
     end
   end
+
+  context "with last updated date in bad format" do
+    setup do
+      message = ONIX::ONIXMessage.new
+      message.parse('test/fixtures/bad_content_date_format.xml')
+
+      @product = message.products.last
+    end
+
+    should "have 1 illustration with nil value for date" do
```

Comment:
s/have 1 illustration with nil value for date/have no last updated date for its illustration